extensions/runtimes/src/index.ts

```diff
@@ -185,8 +208,34 @@ export default async (composer: any): Promise<void> => {
       }
       composer.log('BUILD COMPLETE');
     },
+    installComponent: async (runtimePath: string, packageName: string, version: string): Promise<string> => {
+      // run dotnet install on the project
+      const { stderr: installError, stdout: installOutput } = await execAsync(
+        `npm install --loglevel=error --save ${packageName}${version ? '@' + version : ''}`,
+        {
+          cwd: path.join(runtimePath),
+        }
+      );
+      if (installError) {
+        throw new Error(installError);
+      }
+      return installOutput;
+    },
+    uninstallComponent: async (runtimePath: string, packageName: string): Promise<string> => {
+      // run dotnet install on the project
+      const { stderr: installError, stdout: installOutput } = await execAsync(
+        `npm uninstall --loglevel=error --save ${packageName}`,
```

Comment:
I don't think `--save` is a valid cli flag for uninstall.

Reply:
According to the docs, you do need this to remove it from the dependency list in package.json

https://docs.npmjs.com/cli/v6/commands/npm-uninstall#description